douglas.md

```diff
@@ -0,0 +1,122 @@
+
+Pay attention the next time you are asked for the kind of cookies they're offering to you, when you click somewhere to "choose your preferences". Cookies are usually classified as:
+
+* Strictly necessary cookies - Those are not even needed to grab user's consent to be set and are used to keep basic functionalities working, like keeping your session alive or remembering your cart items on an e-commerce website.
+* Functional cookies - Those are used to remember user's preferences, like language, location, theme, or other settings.
+* Performance cookies - Those are used to track user's behavior on the website, like how long they were on a page, which links they clicked and other parameters that help the website owner to improve the user's experience.
+* Marketing cookies - Those are used to track user's behavior across the web and serve more directionated ads.
+
+Cookies are not essencially bad, as they make everyone's experience using a website better. Or at least they should. 
+
+Following specifications to instruct your browser on how to handle them,  they hold properties like an expiration date (`Expires`), a website address it belongs to (`Domain` and `Path`) and if it is supposed to be transmitted through a `Secure` connection and if it should be accessible by JavaScript or if it is used as `HttpOnly`. [1]
+
+Cookies that are set directly by the domain you're accessing, are called first party cookies, but they can also be generated by secondary calls to third party domains the website uses to render its content; those are called third party cookies.
+
+### First party cookies
+- These are set by the website you're visiting
+- They're used for things like remembering your login status, or your preferences
+
+First party cookies are set after your browser receive the response from the website server when you hit a URL. They come along with this response and contain information about your session. Specially after you login, your browser will receive an instruction to store tokens of some sort, like a JSON Web Token (JWT). Your browser will send the cookie back to the server on every request, so the server knows who you are and what you're allowed to do; providing you Authentication and Authorization, respectively.
+
+Furthermore, developers usually keep track of some actions you took on the website, like links clicked, how long you were on that page, and other parameters useful to **improve your experience** using that website. They want you to keep using it. First party cookies are a very useful tool for everyone and they are not going away.
```

Comment:
Eu acho que como você já falou anteriormente, vale reiterar aqui o relação de 1pc e 3pc com o **domínio**.

Reply:
inclusive cookie enviado na response o JS pode não ser habilitado pra pegar. se for httponly. não lembro se js consegue criar um cookie httponly. vou ver.